### ISSUE TITLE
No such file or directory when project path has space

### ISSUE THREAD

**Issue by jarielbalberona** [cosmtrek/air#546]
I know that having spaces in folder names is weird but I have no problem with this setup with my previous projects and stacks.

/bin/sh: /Users/jariel/Documents/projects/1.: No such file or directory

I've fixed this by removing space in my folder name, from /1. dev/ to /1.dev/.

**Comment by luisgarciaalanis** [comment]
you can also esape the space?

like so -> /1.\ dev

**Comment by jarielbalberona** [comment]
I've no control over the path when I run the "air".

**Comment by mateusfmello** [comment]
My directory structure has spaces because I use client company names to organize files, so some clients have names containing the space character

**Comment by jonasricardo** [comment]
same problem. Any forecast for adjustment?

**Comment by nanvenomous** [comment]
I'm having the same issue **except** my binary file is built as a https://github.com/spf13/cobra command.

my config:
```toml
  bin = "./main serve"
```
From docker container I get the same error message:
```
/bin/sh: line 1: /root/app/main serve: No such file or directory
```

My temporary fix is to cap my version of air (v1.61.7 doesn't have the bug):
```bash
go install github.com/air-verse/air@v1.61.7
```

**Comment by nanvenomous** [comment]
looks like could be related to https://github.com/air-verse/air/pull/749 which was merged recently to add support for spaces in filepaths, but now cannot handle spaces in commands. @pkashin

**Comment by williambao** [comment]
> I'm having the same issue **except** my binary file is built as a https:github.com/spf13/cobra command.
>
> my config:
>
>   bin = "./main serve"
> From docker container I get the same error message:
>
> ```
> /bin/sh: line 1: /root/app/main serve: No such file or directory
> ```
>
> My temporary fix is to cap my version of air (v1.61.7 doesn't have the bug):我的临时解决方法是限制我的空气版本（v1.61.7 没有这个错误）：
>
> go install github.com/air-verse/air@v1.61.7

Yes, I have the same problem. Version 1.62.0 cannot start my app (also using cobra). After downgrading to version 1.61.7, there is no problem.

**Comment by nanvenomous** [comment]
I figured out how to resolve this on later versions of air!
You must use `args_bin` configuration.
```toml
[build]
bin = './main'
args_bin = ['serve']
``` 

which is equivalent to the following shell command:
```bash
./main serve
```

**Comment by xiantang** [comment]
see this issue https://github.com/air-verse/air/issues/824

**Comment by xiantang** [comment]
try latest air version and use `entrypoint` `go1.25.4 install github.com/air-verse/air@v1.63.2`
